modules/rest/src/main/java/org/apache/ignite/internal/rest/cluster/ClusterManagementController.java

```diff
@@ -84,27 +74,15 @@ public CompletableFuture<Void> init(@Body InitCommand initCommand) {
                     initCommand.cmgNodes());
         }
 
-        return validateConfiguration(initCommand.clusterConfiguration())
-                .thenCompose(ignored -> clusterInitializer.initCluster(
-                        initCommand.metaStorageNodes(),
-                        initCommand.cmgNodes(),
-                        initCommand.clusterName(),
-                        initCommand.clusterConfiguration()
-                ))
-                .exceptionally(ex -> {
-                    throw mapException(ex);
-                });
-    }
-
-    private CompletableFuture<Void> validateConfiguration(@Nullable String configuration) {
-        if (configuration != null) {
-            List<ValidationIssue> validationIssues = clusterCfgValidator.validateHocon(configuration);
-            if (!validationIssues.isEmpty()) {
-                return CompletableFuture.failedFuture(new ConfigurationValidationException(validationIssues));
-            }
-        }
-
-        return CompletableFuture.completedFuture(null);
+        //return validateConfiguration(initCommand.clusterConfiguration())
+        return clusterInitializer.initCluster(
+                initCommand.metaStorageNodes(),
+                initCommand.cmgNodes(),
+                initCommand.clusterName(),
+                initCommand.clusterConfiguration()
+        ).exceptionally(ex -> {
+            throw mapException(ex);
```

Comment:
A note for the future: I don't think that we have an adequate mapping for configuration exceptions. Are there any tests for that?